static/src/javascripts/projects/common/modules/commercial/support-utilities.js

```diff
@@ -1,5 +1,40 @@
 // @flow
-const supportContributeURL = 'https://support.theguardian.com/contribute';
-const supportSubscribeURL = 'https://support.theguardian.com/subscribe';
+import {
+    countryCodeToSupportInternationalisationId,
+    getFromStorage,
+} from 'lib/geolocation';
 
-export { supportContributeURL, supportSubscribeURL };
+// Will not change the link if there's no country code in localStorage
+// (i.e. it bypasses the edition fallback of getSync from lib/geolocation)
+const addCountryGroupToSupportLink = (rawUrl: string): string => {
+    const countryCode = getFromStorage();
+    if (countryCode) {
+        const countryGroup = countryCodeToSupportInternationalisationId(
+            countryCode
+        );
+        return rawUrl.replace(
+            /(support.theguardian.com)\/(contribute|subscribe)/,
+            (_, domain, path) =>
+                `${domain}/${countryGroup.toLowerCase()}/${path}`
+        );
+    }
+
+    return rawUrl;
+};
+
+const supportContributeGeoRedirectURL =
+    'https://support.theguardian.com/contribute';
+const supportSubscribeGeoRedirectURL =
+    'https://support.theguardian.com/subscribe';
+const supportContributeLocalURL = (): string =>
```

Comment:
Seeing as this may or may not actually be the local URL, do you think the name should reflect this?

Something like `supportContributeMaybeLocalURL`

Or, make it return `null` if it can't get the local url and export a new function called `supportContributeMaybeLocalURL`

Reply:
good point, I'll just call it `supportContributeURL`